src/dune

```diff
@@ -2,6 +2,6 @@
  (name checkseum)
  (public_name checkseum)
  (libraries optint)
+ (virtual_modules checkseum)
  (wrapped false)
- (flags (:standard -no-keep-locs))
- (modules_without_implementation checkseum))
+ (flags (:standard -no-keep-locs)))
```

Comment:
I'm curious whether the `-no-keep-locs` is really needed? or is it subsumed by dune virtual libraries (see https://github.com/ocaml/dune/issues/1816 -- introduced since 4.06.0 uses `keep-locs` by default and some `cp` was done earlier, resulting in different source directories of the mli and the ml -- https://github.com/mirage/mirage-os-shim/pull/4)

Reply:
yes, that can be removed!

Reply:
Should be noticed by `dune` that this option is not needed anymore if we use `virtual_modules`.